src/test/java/uk/gov/pay/ledger/transactionsummary/dao/TransactionSummaryDaoIT.java

```diff
@@ -145,4 +154,70 @@ public void shouldUpdateFeeForTransactionSummaryCorrectly() {
         assertThat(transactionSummary.get(0).get("total_amount_in_pence"), is(1000L));
         assertThat(transactionSummary.get(0).get("total_fee_in_pence"), is(123L));
     }
+
+    @Test
+    public void verifyMonthlyGatewayPerformanceReportTest() {
+        aTransactionSummaryFixture()
+                .withGatewayAccountId("1")
```

Comment:
Could use random strings like `String gatewayAccountId1 = "account-" + randomAlphanumeric(10);` so multiple tests can avoid using same IDs

Reply:
I'm not sure what this gains as the data is truncated anyways. It could be the case that the `randomAlphanumeric` function returns identical gateway accounts.